### ISSUE TITLE
Images inside PortableText

### ISSUE THREAD

**Issue by Bobetti** [minimiidev/Astro-Sanity-Tailwind-PortableTextWithStyles-Template#1]
Can you make it possible to show images inside blog post text?
Currently no post is published without images or even galleries of images.

**Comment by PaiiNKiilleRD** [comment]
Can you add it? Im not an expert in sanity. But with this template i can customiza the portabletext however i want.

**Comment by Bobetti** [comment]
I created the question because I don't see anywhere on the Internet a way to show images using PortableText.

**Comment by PaiiNKiilleRD** [comment]
Well, have u use this repo?

**Comment by Bobetti** [comment]
Yes, I tested your repo and your PortableTextComponent does not show any images if they are inside post body.
